package.json

```diff
@@ -70,5 +70,6 @@
     "tailwindcss": "^3.4.1",
     "typescript": "^5.4.2",
     "webpack": "^5.91.0"
-  }
+  },
+  "packageManager": "pnpm@8.15.6+sha512.77b89e9be77a2b06ad8f403a19cae5e22976f61023f98ad323d5c30194958ebc02ee0a6ae5d13ee454f6134e4e8caf29a05f0b1a0e1d2b17bca6b6a1f1159f86"
```

Comment:
Wonder if this bit maybe snuck in somehow?

Reply:
@francamps would you be able to remove this addition or close the PR